dyno-demo/src/main/java/com/netflix/dyno/demo/redis/DynoJedisDemo.java

```diff
@@ -223,10 +224,12 @@ public void runScanTest(boolean populateKeys) throws Exception {
 
         CursorBasedResult<String> cbi = null;
         do {
-            cbi = client.dyno_scan(cbi, "DynoClientTest_key-*");
-
-            logger.info("result: " + cbi.getResult().toString());
+			cbi = client.dyno_scan(cbi, 100, "DynoClientTest_key-*");
```

Comment:
there are 500 keys written above, but you are doing count in 100. Is this correct?